### DISCUSSION TITLE
Why is the dialogue formatted in markdown format, but exporting to markdown files is not supported?

### DISCUSSION THREAD

**Discussion by billowssun** [open-webui/open-webui#6501] [General]
![image](https://github.com/user-attachments/assets/ca24ed42-6a04-4d1f-9aad-2053a833ba1f)

**Comment by swatchion** [comment]
In case anyone has the same question, export to markdown can be done easily by use the "Copy" menu, which will copy the whole output token to your copy-paste board, and then you can paste to any text editor to save it as markdown file.

![image](https://github.com/user-attachments/assets/4d59efc2-0447-44de-814e-6abc7174639b)
